src/Controller/RequestController.php

```diff
@@ -43,26 +42,26 @@ public function __construct(array $parameters) {
    *
    * @param \Drupal\graphql\Entity\ServerInterface $graphql_server
    *   The server instance.
-   * @param \GraphQL\Server\OperationParams|\GraphQL\Server\OperationParams[] $operations
+   * @param \Drupal\graphql\Controller\OperationParams|\Drupal\graphql\Controller\OperationParams[] $operations
```

Comment:
we should not change this type, tests are failing. We should keep \GraphQL\Server\OperationParams.